### PR TITLE
[Feature] Manage test modules

### PR DESCRIPTION
**Pull request by fshovchko**
Added add test page and edit test page, where you can create test modules. Result is returned as an object

### CODE REVIEW

File: src/components/FilterData/filterData.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export const filterLevels = ['A1', 'A2', 'B1', 'B2', 'C1', 'C2'];
+export const filterModules = ['Grammar', 'Listening', 'Essay', 'Speaking'];
```
Comment (mtalstykh-exadel) [inline src/components/FilterData/filterData.js]: find a better place for the file. May be src/constants/constants.js

File: src/components/ManageTestModules/ManageGrammar/manageGrammar.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react';
```
Comment (mtalstykh-exadel) [inline src/components/ManageTestModules/ManageGrammar/manageGrammar.js]: start the file with a capital letter

File: src/components/ManageTestModules/ManageGrammar/manageGrammar.scss [hunk 3]
```diff
@@ -0,0 +1,12 @@
+.manage-grammar {
```
Comment (mtalstykh-exadel) [inline src/components/ManageTestModules/ManageGrammar/manageGrammar.scss]: start the file with a capital letter

File: src/components/ManageTestModules/manageModule.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
```
Comment (mtalstykh-exadel) [inline src/components/ManageTestModules/manageModule.js]: you have different names for the file and the folder

**Review by mtalstykh-exadel — APPROVED** [review]

**Review by kseniyaLine — APPROVED** [review]

**Pull request merged**